packages/ui/src/Radio.module.scss

```diff
@@ -0,0 +1,105 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.wrapper {
+  display: inline-flex;
+  // inline-flex changes vertical align compared to display: flex
+  // setting vertical-align allows us to easily use radio button
+  // inline e.g. next to the text
+  // Most common use case will be still the usage in a form where parent
+  // wrapper controls this.
+  vertical-align: bottom;
+  align-items: center;
+  cursor: pointer;
+
+  @include c.typographyBodySmall;
+  // we need to provide spacing between radios if they're in a vertical column
+  margin-bottom: c.$grid;
+
+  // hide native checkbox
+  input {
+    @include h.visuallyHidden;
+  }
+
+  // default visible checkmark
+  .checkmark {
+    width: c.$iconSizeSmall;
+    height: c.$iconSizeSmall;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    border-radius: 50%;
+    // because of ~ and + selectors used below, we need to keep the current order in DOM and
+    // fix it visually with order property
+    order: -1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    // in case we have inline mode, we should have a padding like native radio button has
+    margin-left: c.$grid;
```

Comment:
This also adds margin to default (stacked) radios:
![image](https://user-images.githubusercontent.com/17862968/98538546-36caf000-228b-11eb-9b78-ed9f069b11f6.png)

Reply:
@danad02 should we just forget about it and leave this to the user who will use inline styling? This will be the most common you case.

Reply:
@jvorcak I have to admit that I'm not a fan of the solution you propose. I suggest following:
Add boolean type `inline` prop to `RadioGroup` component. If it's true then we use `flex-direction: row`, otherwise `flex-direction: columns`. WDYT?

Reply:
I thought about it, but I wanted to avoid us having to pass a property that has only to do sth with styling and not the actual functionality, but I guess this is a corner case where we can do that and it'll be easier to use this OOTB. It's fine with me.

Reply:
@jvorcak You don't have to anything with context.

Reply:
Everything you would need to do is:
```
export type RadioGroupProps = RadioGroupCoreProps & {
  ...
  inline?: boolean
} & DataTestProp

/**
 * ### Purpose
 * Forms require input from users. If you need information that can be represented as a choice of multiple values, use input with type 'radio'.
 * You can use RadioGroup in order to group radios buttons into logical cells and provide corresponding error if needed.
 */
export const RadioGroup: FC<RadioGroupProps> = ({
  ...
  inline = false,
}) => {
  const idRef = useRef(uuid())

  return (
    <div className={className} data-test={dataTest}>
      <div
        role="radiogroup"
        data-test="radio-group"
        className={classNames([
          styles.radioGroup,
          radioGroupClassName,
          {
            [styles.inline]: inline,
          },
        ])}
        aria-invalid={!!error}
        aria-errormessage={error && errorId(idRef.current)}
      >
        <RadioGroupContext.Provider value={{ name, error, onChange }}>{children}</RadioGroupContext.Provider>
      </div>
      <Error error={error} className={classNames(styles.errorContainer)} inputId={idRef.current} />
    </div>
  )
}
```

And in `RadioGroup.module.scss` you would add: 
```
.inline {
  flex-direction: row;
}
```

Reply:
But you have to add side margins in children so in the end it would truly require using the context.

Reply:
@danad02 I've done the version with context, pushing soon after screenshots are updated.